.copier-answers.yml

```diff
@@ -14,7 +14,8 @@ odoo_test_flavor: Both
 odoo_version: 15.0
 org_name: Odoo Community Association (OCA)
 org_slug: OCA
-rebel_module_groups: []
+rebel_module_groups:
+- sale_timesheet_rounded
```

Comment:
Hello, I don't remember the exact fault, but AFAIR this was a good idea. Is this a blocking comment?

Reply:
It is, if not justified. The issue is that when we increase the number of tests suites that will run.
So, what is the conflict? Can't we find a better way to prevent that? Eg: disable partly of fully this feature when not needed?

Reply:
@sonhd91 worked on a fix for that in the context of the migration of this module to 17.0; he will backport it and follow-up here

Reply:
@simahawk Here is the fix, please check it out
* https://github.com/OCA/timesheet/pull/660